src/components/UI/Menu/Menu.tsx

```diff
@@ -70,6 +70,7 @@ const Menu: React.SFC<MenuProps> = ({
         transition
         disablePortal={placement === 'top'}
         placement={placement}
+        style={{ zIndex: 1 }}
```

Comment:
Let's update in theme.tsx only to apply global style for this.

Reply:
There is no global class(e.g., Mui-xx) for popper as it is not a part of material ui rather it is a 3rd party library used by react material ui @mdshamoon